### DISCUSSION TITLE
0.3.0 Release Candidate 1

### DISCUSSION THREAD

**Discussion by Siuolplex** [ConsistencyPlus/ConsistencyPlus#27] [General]
Our release candidate for 0.3!

A quick rundown of whats added

-Gates
-Soul Sandstone
-Colored Ice and Glowstone

This is a pre-release. And not all crafting recipes are enabled. Please tell us which ones are missing.

<hr /><em>This discussion was created from the release <a href='https://github.com/ConsistencyPlus/ConsistencyPlus/releases/tag/0.3.0-RC1'>0.3.0 Release Candidate 1</a>.</em>

**Comment by Siuolplex** [comment]
0.3.0 Am I right guys?

**Comment by K0-RR** [comment]
So how many new blocks in this one?

**Comment by K0-RR** [comment]
>not all crafting recipes are enabled. Please tell us which ones are missing

All dyed Glowstone Blocks and Walls.

**Comment by IMS212** [comment]
bad mod - no grass slab

**Reply by K0-RR** [reply]
To be honest, I too would like to see Dirt Slab, especially now when blocks like Ice and Glowstone Fences (the weirdest ones I have even seen but they are nice) were added. There are Obsidian Slabs so dirt ones would be the opposite.

**Reply by TropheusJ** [reply]
:)
